x-pack/plugins/alerting/server/routes/rule/apis/resolve/resolve_rule_route.ts

```diff
@@ -33,19 +33,23 @@ export const resolveRuleRoute = (
     },
     router.handleLegacyErrors(
       verifyAccessAndContext(licenseState, async function (context, req, res) {
-        const rulesClient = (await context.alerting).getRulesClient();
-        const params: ResolveRuleRequestParamsV1 = req.params;
-        const { id } = params;
-        // TODO (http-versioning): Remove this cast, this enables us to move forward
-        // without fixing all of other solution types
-        const rule = (await rulesClient.resolve({
-          id,
-          includeSnoozeData: true,
-        })) as ResolvedRule<RuleParamsV1>;
-        const response: ResolveRuleResponseV1<RuleParamsV1> = {
-          body: transformResolveResponseV1<RuleParamsV1>(rule),
-        };
-        return res.ok(response);
+        try {
```

Comment:
why did we have to add the try/catch?

Reply:
This route was failing because some schemas were missing system actions support so I added this for debugging, but it's not needed anymore. I'll remove.

Reply:
Cool!